src/models/jobProcessor.ts

```diff
@@ -81,4 +84,9 @@ export class JobProcessor {
     this.logger.info({ msg: 'stopping polling' });
     this.isRunning = false;
   }
+
+  private async notifyOnSuccess(jobId: string, taskId: string): Promise<void> {
```

Comment:
split those two functions- move ack() after success case and notify() on finally